index.html

```diff
@@ -127,5 +127,464 @@ <h2 class="uppercase text-4xl text-h2 font-medium">
       </div>
     </section>
```

Comment:
- [ ] The background image is not displayed, to match the Figma design you must use the same background image.

Your design
![sat1](https://user-images.githubusercontent.com/53347966/205434173-5d43dc02-6723-4ab5-a78e-188396894b93.JPG)

Reply:
The background image seems to be working, I have made another commit though.